images/virtualization-artifact/pkg/controller/cvi/internal/source/errors.go

```diff
@@ -65,18 +65,16 @@ func NewVirtualDiskNotReadyError(name string) error {
 	}
 }
 
-type VirtualDiskAttachedToRunningVMError struct {
-	name   string
-	vmName string
+type VirtualDiskEmployedError struct {
+	name string
 }
 
-func (e VirtualDiskAttachedToRunningVMError) Error() string {
-	return fmt.Sprintf("VirtualDisk %q attached to running VirtualMachine %q", e.name, e.vmName)
+func (e VirtualDiskEmployedError) Error() string {
+	return fmt.Sprintf("VirtualDisk %q employed", e.name)
```

Comment:
Let's provide more information about this problem here. Without reading the document, I have no idea what "employed" means in the context of a virtual disk

Reply:
Good idea. Added.